src/oc_erchef/apps/data_collector/rebar.lock

```diff
@@ -1,23 +1,22 @@
-[
- %% these apps are not present in the rebar.conf so
- %% commenting out for now to see what happens.
-%{<<"goldrush">>,
-% {git,"git://github.com/DeadZen/goldrush.git",
-%      {ref,"57e71f2789632a085f0963e1244644fdcb1941b9"}},
-% 1},
-%{<<"ibrowse">>,
-% {git,"git://github.com/chef/ibrowse.git",
-%      {ref,"7b2b0bfd5e15e57706beb2ef0360cdc7f24b37c5"}},
-% 1},
+{"1.1.0",
```

Comment:
Let us create an issue to update the deps for data-collector. And figure out the testing path for it.

Reply:
https://github.com/chef/chef-server/issues/2398

Reply:
Reverted changes to data_collector.